common/app/model/content/Atoms.scala

```diff
@@ -189,3 +201,17 @@ object Quiz extends common.Logging {
     )
   }
 }
+
+object InteractiveAtom {
+  def make(id: String, interactive: atomapi.interactive.InteractiveAtom): InteractiveAtom = {
```

Comment:
@TBonnin, unfortunately I've tried it and it messes with Play JSON's implicit conversion macro, which IIRC uses the `apply` signature to create the `Json.Format[T]`. This is probably why the other atoms used `make`